app/about_us/page.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+export default function about_us() {
```

Comment:
```suggestion
export default function AboutUs() {
```

The naming convention is to use Pascal Case.